core/templates/__base.html

```diff
@@ -131,9 +186,16 @@
     <!-- ===========
     FOOTER
     ============= -->
+
     {% block footer %}
-    {% include 'includes/base_footer.html' %}
+      <div id="footer">
+        <div class="container">
+          <p>All your footer things like logos and such go here.</p>
+          <p>Site created using <a href="https://github.com/open-oni/open-oni">open-oni</a> software, built off the Library of Congress's <a href="https://github.com/LibraryofCongress/chronam">chronam</a>.</p>
+        </div><!-- /container -->
+    </div><!-- /footer -->
```

Comment:
I think this div and the "nav" element above just need one more indentation to even up?